mm/2s2h/Enhancements/Trackers/ItemTracker/ItemTrackerSettings.cpp

```diff
@@ -466,11 +466,14 @@ void SaveItemTrackerLayout() {
     };
 
     auto allConfig = Ship::Context::GetInstance()->GetConfig()->GetNestedJson();
-    if (allConfig.find("Window.ItemTrackerLayout") == allConfig.end()) {
-        allConfig["Window.ItemTrackerLayout"] = nlohmann::json::object();
+    if (allConfig.find("UConfs") == allConfig.end()) {
```

Comment:
Per @garrettjoecox I'm leaning more towards now just having Item Tracker be its own top level block, rather than having a top-level block for miscellaneous things.
```suggestion
    if (allConfig.find("ItemTrackerLayout") == allConfig.end()) {
```